### ISSUE TITLE
Does this repo suppport yolov5 v4 version?

### ISSUE THREAD

**Issue by wxthss82** [marcoslucianops/DeepStream-Yolo#41]
If not, could you tell me the reason? I want to porting to v4 version for performance enhancement and adapt to int8 calibration. Thanks.

**Comment by marcoslucianops** [comment]
Hi, I will work on this update soon. I'm a little busy these days.

**Comment by wxthss82** [comment]
Thank you.

**Comment by rickymedrano** [comment]
@wxthss82 @marcoslucianops 
I'm using Yolov5 v4 and used its models/export.py script to convert it to an onnx file. I then used:
`sudo /usr/src/tensorrt/bin/trtexec --onnx=yolov5s.onnx --fp16 --saveEngine=yolov5ssaved.engine`
to generate an engine file. I then used that engine with your Yolov5 example https://github.com/marcoslucianops/DeepStream-Yolo/blob/master/YOLOv5.md 
and got Deepstream to run and show me FPS results for my model. The only issue is I don't get any bounding box detections. I verified bounding box detections do work for the default yolov5s model from the instructions though. Thought I would let you know @marcoslucianops as it's a good place to start for fully supporting v4.
This was done on a Jetson Nano.
